src/OrchardCore/OrchardCore.Users.Abstractions/Handlers/UpdateUserContext.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Nodes;
+using OrchardCore.Users.Models;
+
+namespace OrchardCore.Users.Handlers
+{
+    /// <summary>
+    /// Represents a <see cref="UserContextBase"/> for updated user roles.
+    /// </summary>
+    public class UpdateUserContext : UserContextBase
+    {
+        /// <summary>
+        /// Creates a new instance of <see cref="UpdateUserContext"/>.
+        /// </summary>
+        /// <param name="user">The <see cref="IUser"/>.</param>
+        /// <param name="loginProvider">The login provider.</param>
+        /// <param name="externalClaims">The user claims.</param>
+        /// <param name="userProperties">The user properties.</param>
+        public UpdateUserContext(IUser user, string loginProvider, IEnumerable<SerializableClaim> externalClaims, JsonObject userProperties) : base(user)
+        {
+            ExternalClaims = externalClaims.AsEnumerable();
+            LoginProvider = loginProvider;
+            UserProperties = userProperties.DeepClone() as JsonObject;
+        }
+
+        /// <summary>
+        /// Gets the login provider.
+        /// </summary>
+        public string LoginProvider { get; }
+
+        /// <summary>
+        /// Gets a list of external claims.
+        /// </summary>
+        public IEnumerable<SerializableClaim> ExternalClaims { get; } = [];
```

Comment:
```suggestion
        public IEnumerable<SerializableClaim> ExternalClaims { get; }
```
Don't initialize it here, it's set in the c'tor. This would initalize the field twice.

Reply:
Hi @giannik , Why you can use `suggestion function`, you are not a member of OC 😳
Show me how it works, please. Please.🫡

Reply:
When you comment on a source code line (or multiple lines), there is a button for in the editor to add a suggestion. See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request (no. 7).

Reply:
Thank you!